03.2_school_prepare_OD_file.R

```diff
@@ -72,44 +85,54 @@ sf11 <- left_join(sf11, lsoa_lookup2, by="lsoa11cd")
 
 # Define phase of education, for 2010 and 2011
 for (i in (10:11) ) {
-  sd <- read_tsv(file.path(path_inputs, "01_raw/02_travel_data/school/NPD_originals_PRIVATE", paste0("SLD_CENSUS_20",i,".txt")))
+  sd <- read_tsv(file.path(path_inputs, "01_raw/02_travel_data/school/lsoa/NPD_originals_PRIVATE", paste0("SLD_CENSUS_20",i,".txt")))
   names(sd) <- sub(paste0("LEA",i,"_"), "", names(sd))
   names(sd) <- tolower(names(sd))
   sd$northing <- as.numeric(sd$northing)
   sd$easting <- as.numeric(sd$easting)
-  sd$pt_girls_old <- rowSums(subset(sd, select=(pt_girls_11:pt_girls_19)))
-  sd$pt_boys_old <- rowSums(subset(sd, select=(pt_boys_11:pt_boys_19)))
-  sd$ft_girls_old <- rowSums(subset(sd, select=(ft_girls_11:ft_girls_19)))
-  sd$ft_boys_old <- rowSums(subset(sd, select=(ft_boys_11:ft_boys_19)))
-  sd$old_perc <- rowSums(subset(sd, select=(pt_girls_old:ft_boys_old))) * 100 / sd$headcount_pupils
-  sd$boarding_perc <- sd$boarders_total * 100 / sd$headcount_pupils
+  
+  sd$pt_girls_11plus <- rowSums(subset(sd, select=(pt_girls_11:pt_girls_19)))
```

Comment:
Again a mix of **tidyverse** and base R code but it works and that's the main thing. FYI I think the following code would work fine and be slightly more concise: 

```
rowSums(select(pt_girls_11:pt_girls_19))
```

Reply:
I tried that just now, but then I get an error message (see below). suggest we leave it as it is for now

sd$pt_girls_11plus2 <- rowSums(select(pt_girls_11:pt_girls_19))
  Error in select(pt_girls_11:pt_girls_19) : object 'pt_girls_11' not found